dbms/src/Storages/KVStore/Decode/PartitionStreams.cpp

```diff
@@ -388,9 +388,16 @@ DM::WriteResult RegionTable::writeBlockByRegion(
 
     reportUpstreamLatency(*data_list_read);
     auto write_result = writeRegionDataToStorage(context, region, *data_list_read, log);
-
+    auto prev_region_size = region->dataSize();
     RemoveRegionCommitCache(region, *data_list_read, lock_region);
-
+    auto new_region_size = region->dataSize();
+    if likely (new_region_size <= prev_region_size)
+    {
+        auto net_write = prev_region_size - new_region_size;
```

Comment:
These names here are strange and misleading, will reconsider

Reply:
I will remove `type_net_write` here, I think the total size of KVStore do better than `type_net_write`